plugins/odf/console-extensions.json

```
@@ -606,6 +606,29 @@
       "required": ["MCG"]
     }
   },
+  {
+    "type": "odf.horizontalNav/tab",
+    "properties": {
+      "id": "iam",
+      "after": "objectbucketclaims",
+      "name": "%plugin__odf-console~IAM%",
+      "contextId": "odf-object-service",
+      "href": "IAM",
```

plz check and follow convention what's used at other places...
```suggestion
      "href": "iam",
```